### PR TITLE
fix: clean code & fix bestiary

### LINKED ISSUE #2164: Wrong CSS class in Bestiary
**Describe the bug**
My silverfish bestiary should not have a max-stat CSS class, because it only 5573/10000

**Profile/Location**
(https://sky.shiiyu.moe/stats/mrdd02/Banana#Bestiary) (Silverfish)

**Additional Info**
-

**Expected behaviour**
The container does not have a max-stat CSS class.

**Screenshots**
image

**Desktop (please complete the following information):**

- OS: Windows 11
- Browser: Chrome
- Version: 120.0.6099.217

**Additional context**
-

### PR DESCRIPTION
**Pull request by DuckySoLucky**
## Description

Fixes #2164 
> Clean code by quite a lot and changed text to show more useful data

## Preview

image

### CODE REVIEW

File: src/constants/bestiary.js [hunk 1]
```diff
@@ -541,12 +541,12 @@ export const BESTIARY = {
         name: "Pigman",
```
Comment (jani270) [inline src/constants/bestiary.js:541]: This looks wrong? They removed the Pigman Besitary

Comment (DuckySoLucky) [reply src/constants/bestiary.js:541]: get a life.

Ik I'm waiting for u to update constants, I'm using https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/blob/prerelease/constants/bestiary.json (old constants were broken I missed smth)

Comment (jani270) [reply src/constants/bestiary.js:541]: > get a life.

What? 

I already sent you a version without it?

Comment (DuckySoLucky) [reply src/constants/bestiary.js:541]: > > get a life.
> 
> What?

You responded liek 10 mins after I created the PR :skull: 

> I already sent you a version without it?

Yeah but it wasn't ifnal version so I ditched it

**Review by metalcupcake5 — APPROVED** [review]

**Pull request merged**